exhale/graph.py

```diff
@@ -3387,20 +3602,42 @@ def gerrymanderNodeFilenames(self):
     def generateViewHierarchies(self):
         '''
         Wrapper method to create the view hierarchies.  Currently it just calls
-        :func:`~exhale.graph.ExhaleRoot.generateClassView` and
+        :func:`~exhale.graph.ExhaleRoot.generatePageView`,
+        :func:`~exhale.graph.ExhaleRoot.generateClassView`, and
         :func:`~exhale.graph.ExhaleRoot.generateDirectoryView` --- if you want to implement
         additional hierarchies, implement the additionaly hierarchy method and call it
         from here.  Then make sure to ``include`` it in
         :func:`~exhale.graph.ExhaleRoot.generateAPIRootBody`.
         '''
+        # gather the page hierarchy data and write it out
+        page_view_data = self.generatePageView()
+        self.writeOutHierarchy({
+            "idx": configs._page_hierarchy_id,
+            "bstrap_data_func_name": configs._bstrap_page_hierarchy_fn_data_name,
+            "file_name": self.page_hierarchy_file,
+            "file_title": "Content",
+            "type": "page"
```

Comment:
Whatever the change is above (`hierarchy` vs `hierarchyType`), lets make `writeOutHierarchy` expect the same thing.

Reply:
Hmm, do you think that `hierarchy_config['type']` isn't explicit enough?

Reply:
Oh yeah that's fine as is, sorry!